humanitec-resource-defs/memorystore/redis/README.md

```diff
@@ -0,0 +1,46 @@
+<!-- BEGIN_TF_DOCS -->
+## Requirements
```

Comment:
Unrelated to this line, could you also add this to https://github.com/humanitec-architecture/resource-packs-gcp/blob/main/README.md?plain=1#L13 similar to how they are listed for AWS?